docs/error-codes.md

```diff
@@ -233,3 +233,7 @@ error and warning codes.
 #### `IL2044`: Could not find any type in namespace 'namespace' specified in 'XML document location'
 
 - The 'XML document location' specifies a namespace 'namespace' but there are no types found in such namespace.
+
+#### `IL2045`: Custom Attribute 'type' is being referenced in code after Custom Attribute type was removed
```

Comment:
I think the wording is a little bit strange here and doesn't tell me much about what should I do to resolve the warning.

Reply:
```suggestion
#### `IL2045`: Attribute type 'type' is being referenced in code but the linker was instructed to remove all instances of this attribute. If the attribute instances are necessary make sure to either remove the linker attribute XML portion which removes the attribute instances, or to override this use the linker XML descriptor to keep the attribute type (which in turn keeps all of its instances).
```

Something along these lines. I know this PR doesn't implement the descriptor part, but we will do that and the warning should be actionable.